docs/source/_templates/data_check_class.rst

```diff
@@ -12,19 +12,6 @@
    .. autoattribute:: name
 
    {% endblock %}
-
```

Comment:
DataCheck subclasses don't have any instance attributes so this section is unnecessary.

Reply:
Love it.  Leaving things cleaner than when you found them is an excellent habit.